AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -662,30 +737,31 @@ public AnalyzeResult GetResult()
 
                     while (!done)
                     {
-                        Thread.Sleep(10);
-                        var current = _metaDataHelper?.Files.Count ?? 0;
+                        Thread.Sleep(100);
```

Comment:
100 seems to be repeated in a couple places, so maybe a constant?

Reply:
Good idea.